decidim-core/app/cells/decidim/nav_links_cell.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
```

Comment:
i would have preffered to skip the new cell entirely ( example use ActionView's render partial, instead of cell rendering ) ... but if you already call this from other cells, I would consider that is ok as it is :)